haya-hayaodeh/Homework-1/fizzbuzz/app.js

```diff
@@ -25,3 +25,19 @@
 // ##### Step 4:
 // Add an additional `else if` clause that logs the string `"fizzbuzz"` if the value being iterated over is divisible by both `3` and `5`.
 
+for (let i = 1; i < 101; i++) {
+  if (i % 3 === 0) {
+    console.log('fizz');
+    } 
+  if (i % 5 === 0) {
```

Comment:
Remember that multiple `if` statements are evaluated independently. For this code, all 3 `if` statements are evaluated each time through the loop, even if the first one is true. Instead, you want to chain them together with `else if`.